wiki/Reporting_Bad_Behaviour/Handling_Foul_Play/en.md

```diff
@@ -0,0 +1,78 @@
+# Handling Foul Play
+_Main page: [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour)._
+
+
+## Why am I restricted?
+A user may get restricted if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
+-   Multi-accounting.
+-   Hacking or cheating (see below).
+
+
+## What is considered multi-accounting?
+Each user may only use **one** account. Creating new accounts is strictly forbidden. First offences will be punished with a silence of varying length depending on the number of accounts made. Users that continue creating new accounts despite previous warnings will be restricted.
+
+
+## What is considered cheating?
+The [rules](/wiki/Rules) that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player is considered cheating.
+
+This is a one-strike rule. Cheaters will be restricted on their first offence for an indefinite amount of time. The team may decide to lift their restriction if they successfully appeal.
+
+
+## I have been restricted for cheating, but I didn't cheat!
+osu! uses automated metrics combined with manual confirmation. If you don't know why you are restricted, contact [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+
+## Prove that I/they were cheating! Tell us how you know!
+The team will not provide any proof. Giving away the countermeasures would make them useless in further detecting cheaters.
```

Comment:
The team will not provide any "proof" regarding restrictions, but all are placed with well founded suspicion. Normal users are extremely unlikely to be targeted by such actions,